assemblies/managing-custom-training-images.adoc

```diff
@@ -5,6 +5,9 @@ ifdef::context[:parent-context: {context}]
 [id="managing-custom-training-images_{context}"]
 = Managing custom training images
 
+To run distributed training jobs, you can use one of the base training images that are installed with {productname-short}, or you can create your own custom training images. 
```

Comment:
I'm not sure about using `installed` here. We do not install the kfto image anywhere, but it is available to be used.
```suggestion
To run distributed training jobs, you can use one of the base training images that are available with {productname-short}, or you can create your own custom training images. 
```

Reply:
Oops, I changed it to "provided" instead of "available" - is that ok or should I change it to "available"?

Reply:
Provided is perfect!